src/Sulu/Bundle/ContentBundle/Controller/SmartContentItemController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContentBundle\Controller;
+
+use Sulu\Component\Rest\RequestParametersTrait;
+use Sulu\Component\Rest\RestController;
+use Sulu\Component\SmartContent\Rest\ItemCollectionRepresentation;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Provides results for smart-content filters.
+ */
+class SmartContentItemController extends RestController
+{
+    use RequestParametersTrait;
+
+    /**
+     * Resolves filter for smart-content UI.
+     *
+     * @param Request $request
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     *
+     * @throws \Sulu\Component\Rest\Exception\MissingParameterException
+     * @throws \Sulu\Component\SmartContent\Exception\DataProviderNotExistsException
+     */
+    public function getItemsAction(Request $request)
```

Comment:
hmm i would mark it as deprecated until 1.2 ... because if "someone" out there using it he has the chance to change it ...